src/components/product/OutOfStock/out-of-stock.scss

```diff
@@ -0,0 +1,51 @@
+@import "src/styles/scaffold";
+
+[data-store-out-of-stock] {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
```

Comment:
```suggestion
```

In this case, we can just remove it. As shown in the image below, the section will occupy the whole space available.  😉 

|flex-start| default (stretch)|
|-|-|
![flex-start](https://user-images.githubusercontent.com/3356699/161094964-76538d9a-3bad-4ef8-a499-eaad445ca199.png)|![default](https://user-images.githubusercontent.com/3356699/161095054-3586997c-ed26-4c19-aeaf-a45e89bc14de.png)